corehq/toggles/__init__.py

```diff
@@ -2484,3 +2473,15 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     help_link='https://confluence.dimagi.com/display/saas/Offer+hosting+and+'
               'sharing+data+files+for+downloading+from+a+secure+dropzone',
 )
+
+REGEX_FIELD_VALIDATION = FrozenPrivilegeToggle(
+    privileges.REGEX_FIELD_VALIDATION,
+    'regex_field_validation',
+    label='Regular Expression Validation for Custom Data Fields',
+    tag=TAG_SOLUTIONS_OPEN,
+    namespaces=[NAMESPACE_DOMAIN],
+    # description="This flag adds the option to specify a regular expression "
```

Comment:
Is this intentional?

Reply:
This was a mistake, and was meant to be uncommented again after I finished the implementation. I have made a commit to rectify this.